src/lesson4/Calculator.java

```diff
@@ -0,0 +1,59 @@
+package lesson4;
+
+import java.util.Scanner;
+
+public class Calculator {
+    static Scanner scanner;
```

Comment:
what is the point of having this variable on class level and with default access modifier?